### PR TITLE
Add docs for BPM

### PR DESCRIPTION
**Pull request by rohitsakala**
[#165019844](https://www.pivotaltracker.com/n/projects/2192232/stories/165019844)

### CODE REVIEW

File: docs/from_bosh_to_kube.md [hunk 1]
```diff
@@ -357,13 +356,38 @@ tags:
 
 ## BPM
 
-### Entrypoint
+The BPM's are converted into containers of `ExtendedStatefulSet's` and `ExtendedJob's` pods. Each `process` in the BPM is run in a single `container` of kubernetes pods. Following subsections will describe about the mapping of BPM configuration into containers.
```
Comment (manno) [inline docs/from_bosh_to_kube.md]:
```suggestion
In a BOSH release some jobs have BPM configuration in `templates/bpm.yml.erb`. These BPM configs are converted into containers. For instance groups of the type `service` the containers are added to the `ExtendedStatefulSet`'s, for `errands` to the `ExtendedJob`'s pods.
Each `process` specified in the BPM configuration is run in a single Kubernetes `container` as part of a pod.

The following subsections describe the mapping of BPM configuration into containers.
```

File: docs/from_bosh_to_kube.md [hunk 2]
```diff
@@ -357,13 +356,38 @@ tags:
...
 
-### Environment
+### Entrypoint & Environment Variables
+
```
Comment (manno) [inline docs/from_bosh_to_kube.md]: These seem to be from https://bosh.io/docs/bpm/config/#process-schema

File: docs/from_bosh_to_kube.md [hunk 3]
```diff
@@ -357,13 +356,38 @@ tags:
...
+`workdir`             | `workingDir`. Not Supported. |
+`hooks`               | `Not Supported` |
+`capabilities`        | `container.SecurityContext.Capabilities`. Not yet supported.  |
+`limits`              | `container.Resources.Limits`. Not yet supported. |
+`ephermeral_disk`     | Not Supported. |
+`persistent_disk`     | Supported. |
+`additional_volumes`  | maps to `VolumeClaimTemplates` which creates Persistent Volume claims in kube cluster. Not yet supported. |
+`unsafe`              | Unmapped to Kubernetes resources. Can be programmatically supported. Not yet supported. |
+
+## Healthchecks
```
Comment (manno) [inline docs/from_bosh_to_kube.md]: Couldn't find Healthcheck in BPM?

```suggestion
## Healthchecks

BPM doesn't provide information for health checks and relies on monit instead. CF-Operator provides health checks via the [BOSHContainerization](https://github.com/cloudfoundry-incubator/cf-operator/blob/2cab6516e8a3fc1f0c814554912449b88dee3101/pkg/bosh/manifest/containerization.go#L7-L13) property in the deployment manifest.
```

File: docs/from_bosh_to_kube.md (hunk 2 above)
Comment (rohitsakala) [reply docs/from_bosh_to_kube.md]: Yes. I tried to map those to kube.

File: docs/from_bosh_to_kube.md (hunk 3 above)
Comment (rohitsakala) [reply docs/from_bosh_to_kube.md]: Yes. As healthcheck is not part of BPM, It is not kept as a subsection under BPM. It is a separate section.

**Pull request merged**